app/src/main/java/com/example/mystore/ui/login/LoginScreen.kt

```diff
@@ -51,126 +65,147 @@ fun LoginScreen(navigateToProductDetailsScreen: () -> Unit = {}) {
     val passwordVisible = rememberSaveable { mutableStateOf(false) }
     var isValidInput by remember { mutableStateOf(false) }
     var isButtonEnabled by remember { mutableStateOf(true) }
-    val mContext = LocalContext.current
-    val coroutineScope = rememberCoroutineScope()
-    var loginViewModel: LoginViewModel = hiltViewModel()
-    val errorMessage = "Your email or password are incorrect"
+    val snackbarHostState = remember { SnackbarHostState() }
+    val loginViewModel: LoginViewModel = hiltViewModel()
     val isValidEmail: (email: String) -> Boolean = { email ->
```

Comment:
Will be cleaner if this was a `String` `extension function` in an `Extensions` file outside of this view.